src/deluge/gui/context_menu/clip_settings/clip_settings.cpp

```diff
@@ -62,6 +70,9 @@ bool ClipSettingsMenu::acceptCurrentOption() {
 			openUI(&launchStyle);
 		}
 		else {
+			// disable shortcut blinking in the rename UI because you don't see
+			// the clip pad on the grid
+			uiTimerManager.unsetTimer(TimerName::SHORTCUT_BLINK);
```

Comment:
isn't there a specific function for clearing the blink state? otherwise I think the pad can get stuck on white.

Reply:
> isn't there a specific function for clearing the blink state? otherwise I think the pad can get stuck on white.

nope for this one it's different. because the blinking uses the function PadLEDs::flashMainPad(x,y) I don't think it can get stuck on white.